ee/clickhouse/queries/test/__snapshots__/test_event_query.ambr

```diff
@@ -74,24 +74,18 @@
     (SELECT id
      FROM person
      WHERE team_id = 2
+       AND id IN
+         (SELECT id
+          FROM person
+          WHERE team_id = 2
+            AND (has(['test'], replaceRegexpAll(JSONExtractRaw(properties, 'name'), '^"|"$', ''))) )
      GROUP BY id
-     HAVING max(is_deleted) = 0) person ON person.id = pdi.person_id
+     HAVING max(is_deleted) = 0
+     AND (has(['test'], replaceRegexpAll(JSONExtractRaw(argMax(person.properties, version), 'name'), '^"|"$', '')))) person ON person.id = pdi.person_id
   WHERE team_id = 2
     AND event = 'viewed'
     AND toTimeZone(timestamp, 'UTC') >= toDateTime(toStartOfDay(toDateTime('2021-05-01 00:00:00', 'UTC')), 'UTC')
     AND toTimeZone(timestamp, 'UTC') <= toDateTime('2021-05-07 23:59:59', 'UTC')
-    AND (pdi.person_id IN
-           (SELECT id
-            FROM person
-            WHERE team_id = 2
-              AND id IN
-                (SELECT id
-                 FROM person
-                 WHERE team_id = 2
-                   AND ((has(['test'], replaceRegexpAll(JSONExtractRaw(properties, 'name'), '^"|"$', '')))) )
-            GROUP BY id
-            HAVING max(is_deleted) = 0
-            AND ((has(['test'], replaceRegexpAll(JSONExtractRaw(argMax(person.properties, version), 'name'), '^"|"$', ''))))))
```

Comment:
Seems correct, but I can't figure out where this change is coming from.

Reply:
I don't know either 🤔 Even though it looks good

Reply:
Oh, that's because of this diff https://github.com/PostHog/posthog/pull/16229/files#diff-81b09b4d64b61fe9e989ab1e49f40a4a7885d38e41df15eac395715da1dc3371
Also leftover from the previous HogQL-based approach, reverted